packages/discovery-provider/src/challenges/listen_streak_endless_challenge.py

```diff
@@ -18,11 +18,10 @@
 env = shared_config["discprov"]["env"]
 
 NUM_DAYS_IN_STREAK = 7
-
-base_timedelta = timedelta(days=1)
-if env == "stage":
-    base_timedelta = timedelta(minutes=1)
-    NUM_DAYS_IN_STREAK = 3
+NUM_HOURS_UNTIL_NEXT_STREAK = 16
```

Comment:
nit: I think RANGE_START & RANGE_END may be a bit more clear. The next streak doesn't actually start in 16hrs, but the var name kinda implies that it does